### PR TITLE
Adding FIPS notice to README

### PR DESCRIPTION
**Pull request by DukeDavis12**
Adding FIPS notice to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -492,4 +492,10 @@ $ cd <fdo-pri-src>/component-samples/demo/device
 $ java -jar device.jar
 ```
 
+## Notice: Use of Bouncy Castle FIPS as Security Provider in FDO Project
 
+FDO project utilizes Bouncy Castle FIPS as its primary security provider. This includes all cryptographic operations within the project.
+
+However, it's important to note that there is one exception to this: the Key Derivation Function (KDF) implementation. For this specific operation, we use an unapproved implementation which follows FDO Spec.
```
Comment (v-vasavi) [inline README.md:499]: The PRI FIDO IOT component uses Bouncy Castle FIPS as the primary security provider for all cryptographic operations within the project with the exception of the KDF.

Comment (v-vasavi) [inline README.md:499]: The KDF implementation is compliant with the FIDO specification and is not based on the Bouncy Castle FIPS.

File: README.md [hunk 2]
```diff
@@ -492,4 +492,10 @@ $ cd <fdo-pri-src>/component-samples/demo/device
...
+
+Aside from the KDF, all other cryptographic operations strictly adhere to the Bouncy Castle APIs.
```
Comment (v-vasavi) [inline README.md:501]: Let us remove this

File: README.md (hunk 1 above)
Comment (DukeDavis12) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (DukeDavis12) [reply README.md]: Updated.

**Review by KiranSukhavasi — APPROVED** [review]

**Review by shrikant1407 — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Pull request merged**